converter/src/main/java/gov/cms/qpp/conversion/decode/SkeletalNameDecoder.java

```diff
@@ -0,0 +1,32 @@
+package gov.cms.qpp.conversion.decode;
+
+import gov.cms.qpp.conversion.Context;
+import gov.cms.qpp.conversion.model.Node;
+
+import java.util.Objects;
+
+import org.jdom2.Element;
+
+public abstract class SkeletalNameDecoder extends QrdaDecoder {
+
+	public static final String NAME = "name";
+
+	private final String name;
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename field "name" to prevent any misunderstanding/clash with field "NAME" defined on line 12. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1845)